backend-dotnet/Meowmentum.Server.Dotnet.Api/Meowmentum.Server.Dotnet.Infrastructure/Implementations/LoginService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Meowmentum.Server.Dotnet.Business.Abstractions;
+using Meowmentum.Server.Dotnet.Core.Entities;
+using Meowmentum.Server.Dotnet.Shared.Requests;
+using Meowmentum.Server.Dotnet.Shared.Results;
+using Microsoft.AspNetCore.Identity;
+
+public class LoginService : ILoginService
+{
+    private readonly UserManager<AppUser> _userManager;
+    private readonly ITokenService _jwtService;
```

Comment:
rename it to smth like _tokenService. Business part doesn't know anything about jwt. It want simply get some token